### ISSUE TITLE
Interaction menu on 1280x1024

### ISSUE THREAD

**Issue by Gwynblade** [acemod/ACE3#2877]
ACE3 Version: 3.3.2

**Mods:**
- `@CBA_A3`
- `@ace`

**Description:**
When you try to detonate any kind of explosives on this resolution, interaction menu dissapears the moment "self actions" icon touches the left side of the screen. Watch the video for better explanation.

**Steps to reproduce:**
https://www.youtube.com/watch?v=gKqPkCgk1Rk

**Comment by Gwynblade** [comment]
Ideally, this would be fixed by getting rid of the two sub-menus - http://imgur.com/uw9UrCw
Going through three sub-menus to detonate an explosive, is really inconvenient when you're setting up an ambush, and need to detonate it fast. Detonating 4 explosives really fast is what nightmares are made of :)

If we could something like this  - http://imgur.com/jAPZbK7
When you select explosives, you can immediately select what to place, what to detonate.
Place/Detonate icons are there just as description, and are non interactable.

Also adding detonate all option would be very useful.

**Comment by esteldunedain** [comment]
Fixed in #3120

**Comment by Gwynblade** [comment]
This is awesome, thanks!

I really hate to do this, but, there are still some issues left on this resolution.

•     2D scopes are cut: https://cloud.githubusercontent.com/assets/1235520/7111788/c7d9dc50-e1c3-11e4-96d3-3c82efb855c9.jpg --- RCO is **not** affected, MRCO and ARCO are.


•     Also, with the recent change in fonts, small interaction menu looks messy and tricky to read: http://imgur.com/TMILuBR. 
Very small menu size is completely unusable: http://imgur.com/uAJiVoA, so I hoped to change it to larger ones.

The problem is, in larger one the menu is cut: http://imgur.com/cMUK28K (if you're using show mouse for interaction). If you're not using that, then the menu disappears if it touches the edge of the screen, like in the video in the first post (just using medical stuff instead)
In largest menu size, the last sub menu is completely hidden: http://imgur.com/xvX3JZq.

Could you maybe bring the submenus bit closer to each other, more place could be gained that way?
http://imgur.com/ifhAGma

Or move the whole thing a bit to the left since a lot of the screen is not used?
http://imgur.com/6PyYcaO


In any case, these are not big issues, so no pressure or anything :)

**Comment by esteldunedain** [comment]
> Could you maybe bring the submenus bit closer to each other, more place could be gained that way?
> http://imgur.com/ifhAGma

Yes, that sounds feasable.

**Comment by Gwynblade** [comment]
Great!

**Comment by esteldunedain** [comment]
Also I didn't confirm it yet, but I think the new default font in Arma makes the small letter size much more usable.

**Comment by Gwynblade** [comment]
Heh, you got me confused there, went to double check, but I'm now certain.

That is the new font in those screenshots. Before the new font, I used the small sized ACE menu, and didn't have the problems with cut off text (with the exception of explosives submenu, which you guys took care off).

But now, due to the font change, small size looks bad, very small is even worse. Other sizes look good, but have cut off text to a varying degree on my resolution.

Basically, BIS screwed me up, and I'm hoping you guys can help :)

**Comment by Mike-MF** [comment]
Closing due to age, inactivity or already solved. May be re-opened if the issue is still present, even better just open a new up-to-date Issue.
